kevm-pyk/src/kevm_pyk/kompile.py

```diff
@@ -173,11 +175,19 @@ def kevm_kompile(
         raise
 
 
-def _lib_ccopts(kernel: Kernel) -> list[str]:
-    ccopts = ['-g', '-std=c++14', '-lff', '-lcryptopp', '-lsecp256k1', '-lssl', '-lcrypto']
+def _lib_ccopts(kernel: Kernel, llvm_kompile_type: LLVMKompileType | None = None) -> list[str]:
```

Comment:
The argument `llvm_kompile_type` is unused here - is that intentional?

Reply:
ah sorry, it was used briefly but then changed the code and this was left in. thanks for the spot.